### DISCUSSION TITLE
any plan to support Admonitions?

### DISCUSSION THREAD

**Discussion by zillionare** [slidevjs/slidev#1216] [Ideas]
I like slidev but seems one important feature is missed -- admonition. Any plan to support this?

Or, can somebody share how to workaround this by using markdown-it and markdown-it-admon plugin? Or, any other solutions?

**Comment by leejoramo-d51** [comment]
I would like admonitions, and other markdown enhancements. I have just started looking around the documentation and found this https://sli.dev/custom/config-vite.html#configure-internal-plugins which references adding markdown-it addons.

**Reply by zillionare** [reply]
Thanks for the hint. Finally I made it work, and share in case anyone else would need:

1. create vite.config.ts under the root of your project
2. add markdown part to the file
```js
import { defineConfig } from 'vite'

export default defineConfig({
    server: {
        fs: {
            strict: false
        }
    },
    slidev: {
        vue: {
            /* vue options */
        },
        markdown: {
            /* markdown-it options */
            markdownItSetup(md) {
                /* custom markdown-it plugins */
                md.use(require('markdown-it-admon'))
            },
        },
  }
})
```
vscode will prompt that some grammar issue. doesn't matter.

**Reply by leejoramo-d51** [reply]
Thanks, that is helpful! I was going to loop back to this in the new year.

**Reply by zillionare** [reply]
pls be notified that the above example will fail in version 0.48, maybe earlier version. It's verified worked under version 0.42.11

**Reply by kermanx** [reply]
Refer to https://github.com/slidevjs/slidev/issues/1325#issuecomment-1958953193.
